PagingWithNetworkSample/app/src/main/java/com/android/example/paging/pagingwithnetwork/reddit/repository/inMemory/byItem/ItemKeyedSubredditDataSource.kt

```diff
@@ -48,9 +49,16 @@ class ItemKeyedSubredditDataSource(
      * <p>
      * See BoundaryCallback example for a more complete example on syncing multiple network states.
      */
-    val networkState = MutableLiveData<NetworkState>()
+    private val _networkState = MutableLiveData<NetworkState>()
```

Comment:
Can you just specfy the type:

val networkState : LiveData<NetworkState> = MutableLiveData<NetworkState>()

Reply:
In that case, the type of `networkState` is `LiveData` so `networkState.postValue` doesn't work because networkState is not a `MutableLiveData`